static/dream_web/index.css

```diff
@@ -1,61 +1,274 @@
-* {
-    font-family: 'Arial';
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700;900&display=swap');
+:root{

```

Comment:
There should be a space between the selector and the opening brackets. See: https://google.github.io/styleguide/htmlcssguide.html

Reply:
Solved